src/Compilers/CSharp/Portable/Symbols/NamespaceSymbol.cs

```diff
@@ -315,48 +315,47 @@ internal NamespaceSymbol GetNestedNamespace(NameSyntax name)
             return null;
         }
 
-        private ImmutableArray<NamedTypeSymbol> TypesMightContainExtensionMethods
+        private ImmutableArray<NamedTypeSymbol> TypesMightContainExtensionMembers
         {
             get
             {
-                var typesWithExtensionMethods = this._lazyTypesMightContainExtensionMethods;
-                if (typesWithExtensionMethods.IsDefault)
+                var typesWithExtensionMembers = this._lazyTypesMightContainExtensionMembers;
+                if (typesWithExtensionMembers.IsDefault)
                 {
-                    this._lazyTypesMightContainExtensionMethods = this.GetTypeMembersUnordered().WhereAsArray(t => t.MightContainExtensionMethods);
-                    typesWithExtensionMethods = this._lazyTypesMightContainExtensionMethods;
+                    this._lazyTypesMightContainExtensionMembers = this.GetTypeMembersUnordered().WhereAsArray(t => t.MightContainExtensionMembers);
+                    typesWithExtensionMembers = this._lazyTypesMightContainExtensionMembers;
                 }
 
-                return typesWithExtensionMethods;
+                return typesWithExtensionMembers;
             }
         }
 
-
         /// <summary>
-        /// Add all extension methods in this namespace to the given list. If name or arity
+        /// Add all extension members in this namespace to the given list. If name or arity
         /// or both are provided, only those extension methods that match are included.
         /// </summary>
-        /// <param name="methods">Methods list</param>
+        /// <param name="members">Members list</param>
         /// <param name="nameOpt">Optional method name</param>
         /// <param name="arity">Method arity</param>
         /// <param name="options">Lookup options</param>
-        internal virtual void GetExtensionMethods(ArrayBuilder<MethodSymbol> methods, string nameOpt, int arity, LookupOptions options)
+        internal virtual void GetExtensionMembers(ArrayBuilder<Symbol> members, string nameOpt, int arity, LookupOptions options)
```

Comment:
In the description, it is probably worth explicitly talking about reduced form of methods returned by this method.